app/components/Icons/CollectionIcon.tsx

```diff
@@ -32,7 +31,7 @@ function ResolvedCollectionIcon({
   if (!collection.icon || collection.icon === "collection") {
     // If the chosen icon color is very dark then we invert it in dark mode
     // otherwise it will be impossible to see against the dark background.
-    const collectionColor = collection.color ?? randomElement(colorPalette);
+    const collectionColor = collection.color ?? colorPalette[0];
```

Comment:
@hmacr random here seemed like a bad default, we still have the "random" color picking logic on a new collection elsewhere

Reply:
random-color shouldn't kick in, no? because a collection icon will have a color unless an emoji was used? 🤔 

When it's an emoji, it follows a similar treatment to "new" collection to use a different color on each render.